### PR TITLE
chore: update branch protection rules, issue management workflow

### PR DESCRIPTION
**Pull request by MIchaelMainer**

### CODE REVIEW

File: .github/policies/resourceManagement.yml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      - addReply:
+          reply: >-
+            Hello @${issueAuthor}
+
+
+            Thank you for reporting your concern. Unfortunately, we in the Graph Explorer team do not have any ownership of the APIs that are causing you issues. If you report this issue in the Microsoft Q&A forum, it will get routed to the appropriate team for them to triage.
+
+            https://aka.ms/askgraph
+
+            This issue will now be closed. If you encounter any issues with Graph Explorer in the future, please feel free to open an issue.
```
Comment (zengin) [inline .github/policies/resourceManagement.yml:150]: Is this text applicable here?

File: .github/policies/resourceManagement.yml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+          days: 14
+      actions:
+      - addLabel:
+          label: Service issue
+      - addReply:
+          reply: >-
+            Hello @${issueAuthor}
+
+
+            Thank you for reporting your concern. Unfortunately, we in the Graph Explorer team do not have any ownership of the APIs that are causing you issues. If you report this issue in the Microsoft Q&A forum, it will get routed to the appropriate team for them to triage.
```
Comment (zengin) [inline .github/policies/resourceManagement.yml:81]: looks like it is copied from the GE repo. Is it applicable here?

**Review by zengin — CHANGES_REQUESTED** [review]

File: .github/policies/resourceManagement.yml (hunk 1 above)
Comment (MIchaelMainer) [reply .github/policies/resourceManagement.yml:150]: THANK YOU!!!! This means I added this elsewhere. Will fix.

File: .github/policies/resourceManagement.yml (hunk 2 above)
Comment (MIchaelMainer) [reply .github/policies/resourceManagement.yml:81]: Nope. I started copying the wrong file.

File: .github/policies/msgraph-metadata-branch-protection.yml [hunk 3]
```diff
@@ -9,3 +9,33 @@ resource: repository
...
+    # Specifies whether this branch can be deleted. boolean
+    allowsDeletions: false
+    # Specifies whether forced pushes are allowed on this branch. boolean
+    allowsForcePushes: false
+    # Specifies whether new commits pushed to the matching branches dismiss pull request review approvals. boolean
+    dismissStaleReviews: true
+    # Specifies whether admins can overwrite branch protection. boolean
+    isAdminEnforced: false
+    # Indicates whether "Require a pull request before merging" is enabled. boolean
+    requiresPullRequestBeforeMerging: true
```
Comment (baywet) [inline .github/policies/msgraph-metadata-branch-protection.yml:25]: This is going to block our currently weekly generation pipeline that relies on pushing directly to master for the time being

**Review by baywet — CHANGES_REQUESTED** [review]

**Review by baywet — APPROVED** [review]

**Pull request merged**